README.md

```diff
@@ -1,167 +1,70 @@
----
-page_type: sample
-products:
-- office-sp
-languages:
-- javascript
-- typescript
-extensions:
-  contentType: samples
-  technologies:
-  - SharePoint Framework
-  platforms:
-  - React
-  createdDate: 6/1/2020 12:00:00 AM
----
-
-copy of: https://github.com/pnp/sp-dev-fx-webparts/blob/master/samples/react-my-groups
-
-# My Groups
-## Update
-- Option to show all groups, this will show 20 groups at the time (with pagination).
-- New link available to redirect at a specific page for the creation of the group.
-- You can choose the number group to display in the setting.
-- If the user have more group, see all button will show. In the setting, a input is available for the see all link.
-- Ability to sort by creation date or alphabitical.
-- French as been add.
-- Available in full-with layout.
-- New List style layout available.
+# My Communities
 
 ## Summary
-
-Using Microsoft Graph, this webpart grabs the Office 365 groups the current user is a member of with links to the groups' SharePoint site.
-
-The webpart has been updated to include a grid style layout, in addition to the compact layout as seen below:
-![Grid Demo](./assets/React-MyGroups_Grid.png)
-
-Compact Layout:
+- The webpart shows all the SharePoint teams sites (groups) that the current user is a member of with links to the groups' SharePoint sites.
+- Able to change the language (English/French) in the property pane.
+- Option to show all groups in another page by entering the url of the see all page in propety pane, this will show 20 groups at the time (with pagination).
```

Comment:
Pls change to: Option to show all groups, this will show 20 groups at the time (with pagination).
And add this new item to the list: Ability to add the url for the See all page.

Reply:
Sure! May I delete the 6th point that I put it in bold to differentiate.  (_If the user have more groups, see all button will show. In the setting, a input is available for the see all link._) I just copied and pasted from old readme  but the see all link will display regardless of the user have more groups or not.